src/lib/dev-environment/dev-environment-lando.js

```diff
@@ -165,3 +167,33 @@ export async function landoExec( instancePath: string, toolName: string, args: A
 
 	await task.run( argv );
 }
+
+/**
+ * Sometimes the proxy network seems to disapper leaving only orphant stopped proxy container.
+ * It seems to happen while restarting/powering off computer. This container would then failed
+ * to start due to missing network.
+ *
+ * This function tries to detect such scenario and remove the orphant. So that regular flow
+ * can safelly add a network and a new proxy container.
+ *
+ * @param {object} lando Bootstrapped Lando object
+ */
+async function ensureNoOrphantProxyContainer( lando ) {
+	const proxyContainerName = lando.config.proxyContainer;
+
+	const docker = lando.engine.docker;
+	const containers = await docker.listContainers( { all: true } );
+	const proxyContainerExists = containers.some( container => container.Names.includes( `/${ proxyContainerName }` ) );
```

Comment:
Not sure this will work, but just asking

```suggestion
	const proxyContainerExists = containers.some( container => container.Names.includes( proxyContainerName ) );
```

Reply:
Nah, I tried that and it didn't work. The names returned have some sort of namespaces so `/` for no namespace.